website/project/model.py

```diff
@@ -860,6 +860,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     primary_institution = fields.ForeignField('institution')
 
     affiliated_institutions = fields.ForeignField('institution', list=True)
+    visit = fields.IntegerField(default=0)
```

Comment:
Do we want to keep track of visits on all nodes, or just visits to the materials submitted to conferences?